web/modules/custom/joinup_user/config/install/message.template.og_membership_role_change.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies: {  }
+template: og_membership_role_change
+label: 'Group membership role change'
+description: 'Message sent when a user receives a new role inside a collection or solution.'
+text:
+  -
+    value: '<p>Your role has been changed to @membership:roles</p>'
+    format: full_html
+  -
+    value: '<p>@actor:full_name has changed your role in @group:bundle "@group:title" to @membership:roles.</p><p>If you think this action is not clear or not due, please contact Joinup Support.</p>'
```

Comment:
According to the ticket, the last part, 'please contact Joinup Support', should contain a link to the support page (contact form). This link should be the 'Joinup Support' phrase.